src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
 
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
+        if ($container->has('profiler')) {
```

Comment:
This looks overkill to me. Isn't there any other solution? In other extensions, for example in [these lines](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L371-378) of `FrameworkExtension`, they use the same code as here:

```php
```